README.md

```diff
@@ -57,6 +57,9 @@ For other languages, there are third-party generators available:
 
 |    Language    | Clients | Servers | Repository |
 |----------------|---------|---------|------------|
+| **Go**         |    ✓    |    ✓    | Official support
```

Comment:
Could you remove the official Go and Python entries since the sentences above the table clarifies that support? If you still want to reference issue#185 there, that seems fine to me.

Reply:
I found adding to the table clearer, but I've moved it nevertheless :)

Feel free to make any edits you want.